packages/sdk/typescript/human-protocol-sdk/src/kvstore.ts

```diff
@@ -195,7 +195,7 @@ export class KVStoreClient {
    * const signer = new Wallet(privateKey, provider);
    * const kvstoreClient = await KVStoreClient.build(signer);
    *
-   * const keys = ['Role', 'Webhook_url'];
+   * const keys = ['Role', 'WebhookUrl'];
```

Comment:
Use always same case to avoid case sensitive errors

```suggestion
   * const keys = ['role', 'webhookUrl'];
```